scripts/lib/libvirt/compute-config

```diff
@@ -13,24 +13,20 @@ def main():
                         (when this script is called within a loop, this \
                         is usually an incremented number)")
     parser.add_argument("macaddress", help="MAC Address")
-    parser.add_argument("controller_raid_volumes", type=int,
-                        help="Number of disks for RAID on controller node")
+    parser.add_argument("raid_volumes", type=int,
+                        help="Number of disks for RAID on node")
     parser.add_argument("cephvolumenumber", type=int,
                         help="Number of Ceph Volumes")
     parser.add_argument("drbdserial", help="DRBD Volume Serial")
-    parser.add_argument("computenodememory", type=int,
-                        help="Compute Node Memory (kB)")
-    parser.add_argument("controllernodememory", type=int,
-                        help="Controller Node Memory (kB)")
+    parser.add_argument("nodememory", type=int,
+                        help="Node Memory (kB)")
```

Comment:
@aplanas the call only creates one node config, so we just pass in the "current node's" setting, and determine how the settings are supposed outside the script